stdlib/jsonutils/src/test/java/org/ballerinalang/stdlib/jsonutils/JsonUtilsTest.java

```diff
@@ -179,9 +179,9 @@ public void testComplexXMLtoJson() {
                         "\"Count\":\"7\"}}, \"\\n        \"]}, \"\\n        \", {\"Address\":[\"\\n            \", " +
                         "{\"StreetAddress\":\"20, Palm grove, Colombo 3\"}, \"\\n            \", " +
                         "{\"City\":\"Colombo\"}, \"\\n            \", {\"Zip\":\"00300\"}, \"\\n            \", " +
-                        "{\"Country\":\"LK\"}, \"\\n        \"], \"@xmlns:\":\"\", \"@\":\"\"}, \"\\n    \"], " +
-                        "\"@xmlns:ns\":\"ns.com\", \"@xmlns:\":\"example.com\", \"@attr\":\"attr-val\", " +
-                        "\"@ns:attr\":\"ns-attr-val\", \"@ns\":\"ns.com\", \"@\":\"example.com\"}}");
+                        "{\"Country\":\"LK\"}, \"\\n        \"], \"@xmlns\":\"\"}, \"\\n    \"], " +
+                        "\"@xmlns:ns\":\"ns.com\", \"@xmlns\":\"example.com\", \"@attr\":\"attr-val\", " +
+                        "\"@ns:attr\":\"ns-attr-val\", \"@ns\":\"ns.com\"}}");
```

Comment:
Since this test is failing due to the changes, I have replaced the expected output with current output. @rdhananjaya can you have a look whether this is correct or not?

Reply:
Looks like we had a bug and with this change, it's fixed.

Reply:
Ack. Thanks.